.github/workflows/e2e.yml

```diff
@@ -28,30 +28,53 @@ jobs:
         with:
           submodules: true
           path: vip-go-mu-plugins
+
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
           node-version: '16'
+          cache: npm
+          cache-dependency-path: vip-go-mu-plugins/package-lock.json
+
       - name: Install dependencies
-        run: cd ./vip-go-mu-plugins && npm install
-      - name: Install VIP Cli
-        run: npm install --prefix=$HOME/.local -g @automattic/vip
+        run: npm ci
+        working-directory: vip-go-mu-plugins
+
+      - name: Install VIP CLI
+        run: |
+          npm install --prefix=$HOME/.local -g @automattic/vip
+          cd $HOME/.local/lib/node_modules/@automattic/vip && npm i colors@1.4.1 --force
+
+      - name: Determine WP version
+        id: wpver
+        run: |
+          if [ -z "${{ github.event.inputs.wpversion }}" ]; then
+            echo ::set-output name=wpversion::latest
+          else
+            echo ::set-output name=wpversion::${{ github.event.inputs.wpversion }}
+          fi
```

Comment:
For pull requests, `github.event.inputs.wpversion` is empty; this makes `setup-e2e-env` complain.